app/views/documents/show/_history_tab.html.erb

```diff
@@ -1,5 +1,12 @@
+<% displayed_edition_number = nil %>
+
 <% @document.timeline_entries.includes(:user).order(created_at: :desc).each do |entry| %>
   <div class="timeline-entry govuk-!-margin-bottom-6">
+    <% if displayed_edition_number != entry.edition_number %>
+      <h3 class="govuk-heading-m"><%= entry.edition_number.ordinalize %> edition</h3>
```

Comment:
Should this go through the translations file?

Reply:
👍 done